jabref.install4j

```diff
@@ -561,6 +564,30 @@ return console.askOkCancel(message, true);
                     </serializedBean>
                     <condition>!(Util.hasFullAdminRights() || Util.isAdminGroup())</condition>
                   </action>
+                  <action name="" id="352" customizedId="" beanClass="com.install4j.runtime.beans.actions.registry.SetRegistryValueAction" enabled="true" commentSet="false" comment="" actionElevationType="elevated" rollbackBarrier="false" rollbackBarrierExitCode="0" multiExec="false" failureStrategy="1" errorMessage="">
+                    <serializedBean>
+                      <java class="java.beans.XMLDecoder">
+                        <object class="com.install4j.runtime.beans.actions.registry.SetRegistryValueAction">
+                          <void property="keyName">
+                            <string>SOFTWARE\Mozilla\NativeMessagingHosts</string>
+                          </void>
+                          <void property="registryRoot">
+                            <object class="java.lang.Enum" method="valueOf">
+                              <class>com.install4j.api.windows.RegistryRoot</class>
+                              <string>HKEY_LOCAL_MACHINE</string>
+                            </object>
+                          </void>
+                          <void property="value">
+                            <string>/ve /d "${installer:sys.installationDir}\jabref.json" /f</string>
```

Comment:
@koppor This generates a key with a value `/ve /d "C:\Program Files\JabRef\jabref.json" /f`. However, there actually should be a folder `org.jabref.jabref` with key `Default` and only the path as value. 
![image](https://user-images.githubusercontent.com/5037600/38522071-c39aa4ee-3c47-11e8-81b5-5605ae1ae9f7.png)

It would be nice if you could fix this since this is the last issue that currently prevents merging this PR. Thanks!
See https://github.com/JabRef/JabFox/issues/50.

Reply:
Fixed. I was in a hurry when copy'n'pasting your requirements.